objects/domain-crawled/definition.json

```diff
@@ -0,0 +1,39 @@
+{
+  "required": [
+    "domain"
+  ],
+  "attributes": {
+    "text": {
+      "description": "A description of the tuple",
+      "disable_correlation": true,
+      "ui-priority": 1,
+      "misp-attribute": "text",
+      "recommended": false
+    },
+    "domain": {
+      "description": "Domain name",
+      "categories": [
+        "Network activity",
+        "External analysis"
+      ],
+      "ui-priority": 1,
+      "misp-attribute": "domain",
+      "multiple": true
```

Comment:
Based on our discussion on Friday, I don't think the domain should be a multiple.

Reply:
Good Catch ! Fixed with #233